Add list of additional package installs (#499)
stackhpc/ansible-slurm-appliance: docs/operations.md

Deploying the additional nodes and applying these changes requires rerunning both Terraform and the Ansible site.yml playbook - follow [Deploying a Cluster](#Deploying-a-Cluster).

# Adding Additional Packages
By default, the following utility packages are installed during build:
- htop
- nano
- screen
- tmux
- wget
- bind-utils
- net-tools
- postfix
- git
- latest python version for system (3.6 for for Rocky 8.9 and 3.12 for Rocky 9.4)

> Additional packages from any DNF repositories which are enabled during build (which always includes EPEL, PowerTools and OpenHPC) can be added to the image by defining a list `appliances_other_extra_package` (defaulted to the empty list in the common environment) in e.g. `environments/$SITE_ENV/inventory/group_vars/all/defaults.yml`. For example:

Reviewer: ```suggestion

Additional packages from any DNF repositories which are enabled during build (which always includes EPEL, PowerTools and OpenHPC) can be added to the image by defining a list `appliances_other_extra_packages` (defaulted to the empty list in the common environment) in e.g. `environments/$SITE_ENV/inventory/group_vars/all/defaults.yml`. For example:
```

>     # environments/foo-base/inventory/group_vars/all/defaults.yml:

Reviewer: ```suggestion
```yaml
    # environments/foo-base/inventory/group_vars/all/defaults.yml:
```

    appliances_other_extra_package:
    - somepackage

>     - anotherpackage

Reviewer: ```suggestion
    - anotherpackage
```
```

The packages available from the OpenHPC repos are described in Appendix E of the OpenHPC installation guide (linked from the [OpenHPC releases page](https://github.com/openhpc/ohpc/releases/)). Note "user-facing" OpenHPC packages such as compilers, mpi libraries etc. include corresponding `lmod` modules.

> If you wish to install packages during runtime, the `site.yml` playbook should be run `appliances_packages_during_configure` overriden to `true` and `cluster` should be added as a child of the `dnf_repos` group in order to temporarily re-enabled DNF repositories during runtime (WARNING: this should only be done if using an unauthenticated local Pulp server. If using StackHPC Ark directly, doing this WILL leak credentials to users).

Reviewer: ```suggestion
If you wish to install packages during runtime, the `site.yml` playbook should be run with `appliances_packages_during_configure` overriden to `true` and `cluster` should be added as a child of the `dnf_repos` group in order to temporarily re-enable DNF repositories during runtime (WARNING: this should only be done if using an unauthenticated local Pulp server. If using StackHPC Ark directly, doing this WILL leak credentials to users).
```
